<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>easyStarter</class>
 <widget class="QMainWindow" name="easyStarter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Easy Starter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_title">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Galaxy Simulation Easy Starter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="parameters">
      <property name="title">
       <string>parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="angle_box">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="elev_label">
             <property name="text">
              <string>elev : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="elev_slider">
             <property name="maximum">
              <number>3600</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="azim_label">
             <property name="text">
              <string>azim : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="azim_slider">
             <property name="maximum">
              <number>3600</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="frame_number_box">
         <item>
          <widget class="QLabel" name="frame_number_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Frame :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="frame_number_slider">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="G2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="theta2_label">
           <property name="text">
            <string>theta2 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="theta2_slider">
           <property name="maximum">
            <number>1800</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="phi2_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>phi2 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="phi2_slider">
           <property name="maximum">
            <number>3600</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="Others">
         <item>
          <widget class="QLabel" name="mratio_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Mass Ratio :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="mratio_slider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="peri_label">
           <property name="text">
            <string>Peri :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="peri_slider">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="G1">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="theta1_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>theta1 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="theta1_slider">
           <property name="maximum">
            <number>1800</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="phi1_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>phi1 : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="phi1_slider">
           <property name="maximum">
            <number>3600</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="img_horizon">
      <item>
       <widget class="QLabel" name="img_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="text">
         <string>No img</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="button_horizon_2">
      <item>
       <widget class="QPushButton" name="camera_btn">
        <property name="text">
         <string>카메라 적용</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="main_btn">
        <property name="text">
         <string>main.py 실행</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_btn">
        <property name="text">
         <string>세팅값 보기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="movie_btn">
        <property name="text">
         <string>동영상 만들기</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>고급 설정</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>도움말</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
